templates/partners/partial/_partners-logos.html

```diff
@@ -0,0 +1,20 @@
+<section class="p-section">
+  <div class="row p-block">
+    <hr class="p-rule">
+    <h2>{{ title }}</h2>
+  </div>
+  <div class="row">
+    <div class="p-logo-section">
+      <div class="p-logo-section__items"> 
+        {% for partner in partners %}
+            <div class="p-logo-section__item">
+              <img class="p-logo-section__logo" src="{{ partner.logo }}" alt="{{ partner.slug }}">
+            </div>
+        {% endfor %}
+      </div>  
+    </div>
+    <div class="col-6 col-start-large-7 col-medium-3 col-start-medium-4">
+      <p><a href="{{ link_href }}">{{ link_text }}</a></p>
+    </div>
+  </div>
+</section>
```

Comment:
@mtruj013 What do you think of abstracting common patterns into partials for ease of use and consistency?

Reply:
On the design yes but Lyubo wanted it like that on another page so I just kept it for consistency